src/app_config.cpp

```diff
@@ -232,6 +234,23 @@ ConfigOpt *opts[] =
 ConfigJson user_config(opts, sizeof(opts) / sizeof(opts[0]), EEPROM_SIZE, CONFIG_OFFSET);
 ConfigJson factory_config(opts, sizeof(opts) / sizeof(opts[0]), EEPROM_SIZE, FACTORY_OFFSET);
 
+uint32_t
+getConfigVersion() {
+  return config_version;
+}
+
+void 
+incrementConfig() {
```

Comment:
Should be intenal and also use snake case as the other functions

Reply:
but increment is called from elsewhere (monitor). How to make it internal ?

Reply:
If you switch to using the boot event it can be kept internal.

Reply:
I am resolving it as resolving the other one will also resolve it.